yt_dlp/YoutubeDL.py

```diff
@@ -2777,7 +2777,7 @@ def print_optional(field):
         if info_dict.get('requested_formats') is not None:
             # For RTMP URLs, also include the playpath
             info_dict['urls'] = '\n'.join(f['url'] + f.get('play_path', '') for f in info_dict['requested_formats'])
-        elif 'url' in info_dict:
+        elif info_dict.get('url'):
```

Comment:
This also changes the behavior if `info_dict['url']` is an empty string; intended?

Reply:
It should always be a valid URL if present. So not an issue